### PR TITLE
Add browserify

### PR DESCRIPTION
**Pull request by juliangruber**
cc @yoshuawuyts

### CODE REVIEW

File: lib/transform.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict'
```
Comment (juliangruber) [inline lib/transform.js]: this is required for the bindings module to work with browserify. Should go into a module and use the AST if we decide we want this approach.

Comment (yoshuawuyts) [inline lib/transform.js]: I'm happy to split this into a package if you want - what do you reckon would be a good name?

Comment (mafintosh) [inline lib/transform.js]: bindingify! (no idea what this does though lol)

**Comment by juliangruber** [comment]
> (no idea what this does though lol)

The bindings module needs to know the `__dirname` of the module calling it. It currently uses Error stack traces to get that, but it's way cleaner to just pass the `__dirname` as an argument instead. And, that works with browserify.

**Comment by juliangruber** [comment]
Simply enabling source maps doesn't do that trick either.

**Comment by juliangruber** [comment]
Cool, @yoshuawuyts, are you taking over on the module?

**Comment by yoshuawuyts** [comment]
@juliangruber yup, happy to do it - I propose we merge this as-is and then replace it with an external package in a separate PR :sparkles:

**Comment by yoshuawuyts** [comment]
alright merging :sparkles:

**Pull request merged**